app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str):
+    file1 = command.split()[1]
+    file2 = command.split()[2]
+    if file1 == file2:
```

Comment:
unnecessary condition.

make a check, if they are not equal, then make a record